src/test/java/com/wojciechbarwinski/demo/epic_board_games_shop/EpicBoardGamesShopApplicationTests.java

```diff
@@ -2,8 +2,10 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
 
 @SpringBootTest
+@ActiveProfiles("local")
```

Comment:
Na testy integracyjne robi się dedykowany profil "integration" ;)  on może, ale nie musi być zbliżony do "local"

Reply:
mi tu chodziło tylko o to by test korzystał z 2 propertisów gdzie BD jest resetowana. Tylko tyle

Reply:
Nieważne :P "local" to jest local i służy do lokalnego odpalenia aplikacji, a do testów służy "integration" - tak się przyjęło i akurat tego staramy się trzymać na sztywno ;)